src/cli/backup/exchange_integration_test.go

```diff
@@ -291,13 +291,16 @@ func (suite *PreparedBackupExchangeIntegrationSuite) SetupSuite() {
 
 		switch set {
 		case email:
-			scopes = sel.MailFolders([]string{suite.m365UserID}, []string{exchange.DefaultMailFolder})
+			scopes = sel.MailFolders([]string{suite.m365UserID}, []string{exchange.DefaultMailFolder}, selectors.PrefixMatch())
```

Comment:
hmmm so actually looking at this, we might need exact matches in cases like these. I believe GraphConnector is currently doing exact matches with the folder names (that's likely why we need to pass the full path of a subfolder in to get it to back it up), but changing to a prefix match is not equivalent in this case. Here we actually do just want to get the folder mentioned, but none of its subfolders. If we start pulling in subfolders via prefix matching I worry we may eventually time out backing up contacts because technically all contact folders are a child of the root contact folder

We do have some path munging in there for contacts folders I think (iirc they're flattened, which may need revisiting...?), but it's probably best not to rely on something like that

Reply:
I don't think it's a cause for concern.  In researching these changes the folders we compare are all the first children of the root.  For example, here's the iteration for contacts for the noted test (first word is the category, second is the directory).  Same behavior goes for email and events.
```
contacts Contacts
contacts Corso_Restore_03-Nov-2022_20-39-32.251826
contacts Corso_Restore_03-Nov-2022_20-45-25.710348
contacts Corso_Restore_03-Nov-2022_21-09-07.307926
contacts Corso_Restore_03-Nov-2022_21-10-26.431697
contacts Corso_Restore_03-Nov-2022_21-05-14.498555
contacts Corso_Restore_03-Nov-2022_21-05-23.622714
contacts TrialSubFolder
contacts Corso_Restore_03-Nov-2022_20-37-32.867436
contacts TrialFolder
```